src/jobs/hadolint.yml

```diff
@@ -47,6 +47,14 @@ parameters:
       `docker.io,my-company.com:5000`); if set, return an error if
       Dockerfiles use any images from registries not included in this list
 
+  failure-threshold:
+    type: enum
+    default: "info"
+    description: >
```

Comment:
based on our conversation about how the failure-threshold works, does it make more sense to categorize it as a failure based on any enum match (rather than a threshold), since equal to above doesn't really make sense